Dockerfile

```diff
@@ -61,6 +61,9 @@ COPY tests /app/tests
 ENV PYTHONPATH=.
 COPY bin/startup.sh /app/startup.sh
 COPY redis.conf /app/redis.conf
+
+# Set the SWEEP_VERSION environment variable to the current date and time during image build
+ENV SWEEP_VERSION=$(date +%Y%m%d%H%M)
```

Comment:
buildx failed with: ERROR: failed to solve: Syntax error - can't find = in "+%Y%m%d%H%M)". Must be of the form: name=value